.php_cs

```diff
@@ -74,6 +17,9 @@ return PhpCsFixer\Config::create()
         'phpdoc_annotation_without_dot' => false,
         'phpdoc_no_alias_tag' => false,
         'space_after_semicolon' => false,
+        'yoda_style' => false,
+        'modernize_types_casting' => false,
```

Comment:
you disabled this but some changes here is related to this rule right? `(int)` So should it be enabled or does it then change a lot more code?

Reply:
Ah, sorry, my mistake, I actually fixed code according to this rule and commented [here](#discussion_r144227994).
Removed it from .php_cs, updated PR.

Reply:
+1